### DISCUSSION TITLE
Discussions re-organisation

### DISCUSSION THREAD

**Discussion by Daeraxa** [pulsar-edit/.github#172] [Announcements]
As a follow on from ["Reorganising our discussions"](https://github.com/orgs/pulsar-edit/discussions/168) you may notice that the discussion sections have been re-organised into sections so that (hopefully) it is a little more obvious where to go to discuss your latest Pulsar project or ask for help in getting something to work.

A big change is making the development category more prominent, after all that is what Pulsar is all about.

I'm hoping this improves the experience somewhat but very open to any feedback on the changes - just open a topic in the `Lounge > Discussions Meta` category.

**Comment by confused-Techie** [comment]
Just wanna say the reorganization looks fantastic!

**Comment by gauravweb99** [comment]
Super 😊👍

**Comment by CyberDreamCatcher** [comment]
Hello everyone! I very like your decision to support Atom text editor. I wanna get some answers to my questions:
1. Pulsar text editor is absolutely free? Do pulsar require any money?
2. Why do pulsar for windows don’t get certification of safety? 
3. Do pulsar collect my information like atom telemetry or something else? Can I be sure that no one is following or watching me?

**Reply by Daeraxa** [reply]
This is the wrong topic and has already been answered on Discord
